### ISSUE TITLE
Missing support for the following antennas

### ISSUE THREAD

**Issue by ChubbyTurtle101** [DRVeyl/RealAntennas#17]
Missing one Squad antenna and the antenna from a couple fairly popular mods that are used with RO and non-RO packs.  The recent release of Near Future Exploration looks like some interesting game play with RealAntennas with RSS and planet packs like OPM, etc.

**Squad:**
Communotron DTS-M1

**Near Future Exploration:**
AX-30 High-Gain Micro Antenna
AX-4 Pointable Helical Antenna
AX-5 Aerial Micro Antenna
D-2 Spot Antenna
D-50 Large Spot Antenna
DR-1 High gain Antenna
DR-3 Deployable High Gain Antenna
F-DA Direct Antenna Feed
F-RA Relay Antenna Feed
PH-1 Phased Array Antenna Element
PH-2 Phased Array Antenna Element
PH-3 Phased Array Antenna Element
RA-00-2 Micro-Relay Antenna
RA-0-8 Relay Antenna
RA-SB Advanced Relay Antenna
RA-X1 Phased Relay Antenna
RA-X2 Phased Relay Antenna
RA-X3 Phased Relay Antenna
RFL-1 Dish Reflector
RFL-100 Giant Dish Reflector 
RFL-2 Medium Dish Reflector 
RFL-2000 Dish Reflector Array
RFL-3 Dish Reflector Array
RFL-50 Large Dish Reflector

**ReStock Plus**
Communotron DTS-J1
Communotron HG-61
HG-20 High Gain Antenna

**JX2Antenna**
JU1 Medium Deployable Antenna
JW1 Medium Deployable Antenna
JX2 Large Deployable Antenna

**Comment by DRVeyl** [comment]
The DTS-M1 is a bit of a snub.  The model itself is a bit weird, and we're not sure what to properly assign as an effective area.

For the rest, PRs are welcome, and should follow the format of the other patches in the Parts folder.

**Comment by bofhgr** [comment]
speaking of support for antennas, as far as I could tell, the Deploy-able Ground Experiments from the DLC are not supported - can these be added to the supported list as well? 

Sorry for hijacking this issue, but it seemed fitting with my question, rather than opening a new one, and I couldn't find a thread on the forums for the mod to ask this question. thanks

**Comment by DRVeyl** [comment]
I'm not familiar with those, but if they are parts with antennas (the stock ModuleDataTransmitter), then sure.  As mentioned above, PR's welcome!

**Comment by baldamundo** [comment]
On a related note, what happens to unpatched antennae when running this mod? Do they continue to work as normal by the stock system? Do they not work at all?

And is there a guide somewhere on how to create patches for RealAntenna? Have been using Tantares & CoatlAerospace/ProbesPlus extensively so may get round to making patches for them myself if it's not too difficult.

**Comment by DRVeyl** [comment]
Think I missed that message with this issue being somewhat necro'd.

Unpatched antennas essentially become paperweights.  Important portions of CommNet have been patched so that your vessel only selects from RA-configured antennas, both for creating the comm links and for getting the antenna for science transmission.  If you somehow managed to force an experiment info into an unpatched antenna and asked it to transmit through its PAW, that might "work" but this would be unintended at best (and I don't know how you might cause this.)

There is no guide per-se, but the files in the Patches folder pretty much make a template.  For an omnidirectional antenna, I suggest setting `referenceGain` to 3 or less.  For a directional antenna, the easiest way is to determine the `antennaDiameter` field.  For a round dish, this is just the diameter of that dish.  For planar arrays or slightly more exotic shapes, the best first-order approximation is to just measure the overall area of the reflector, and determine the diameter of a circle that has the same area.

(Sure, that would be slightly wrong for a rectangular planar array, especially a very wide one.  But RA doesn't model non-circular antenna patterns.  Although it certainly could, there is no practical way to rotate, let alone physically point, your antenna in-game, so spherical cows are the order of the day.)

**Comment by Bornholio** [comment]
@DRVeyl  If you have time can you give a part config walk through? I'll write some configs and a Wiki page after I figure it out.

**Comment by Bornholio** [comment]
https://github.com/DRVeyl/RealAntennas/wiki/Part-Configs

**Comment by Bornholio** [comment]
closed by #40

**Comment by Bornholio** [comment]
coatl antennas to add support for 
antenna_cone_toggle
ca_ant_gps
antenna_quetzal
antenna_tv
dish_deploy_S
dish_deploy_S2
dish_hera
dish_L
dish_quetzal
dish_S
dish_tatsujin
dish_xihe

ca_argo-mk2-mast
ca_argo-mk2-hga
ca_argo-mk3-hga
ca_argo-mk4-mast
ca_argo-mk4-hga

:::GroundOps
ca_landv_omni  3dB
ca_landv_hga (has dish settings)
ca_landv_orbiter_HGA make same as ca_landv_hga

Missing more??? 43 name = ModuleDataTransmitter in mod

**Comment by austinjames314** [comment]
> I'm not familiar with those, but if they are parts with antennas (the stock ModuleDataTransmitter), then sure. As mentioned above, PR's welcome!

I've come trying to find a solution to the same problem. The issue is the Deployed Central Station part, featured at the center of this screenshot:
https://cdn.discordapp.com/attachments/410362165795749889/789022921749561364/unknown.png

All the deployed parts in the foreground are stock BTW.

I'm guessing it's also going to be a problem for the DeployedSatDish, when I get that part unlocked. Neither of these parts use ModuleDataTransmitter. The former uses ModuleGroundExpControl and the latter ModuleGroundCommsPart, both of which include values such as:

antennaType = DIRECT		
antennaPower = 500000
antennaCombinable = True
antennaCombinableExponent = 1

Basically I can't figure out how to get the surface experiments to connect to anything.

**Comment by DRVeyl** [comment]
Breaking Ground support will be tracked here: <https://github.com/DRVeyl/RealAntennas/issues/41>
